### PR TITLE
answers: Remove singleton data holder DATA & adapt

### PR DESCRIPTION
**Pull request by meetmangukiya**
Closes https://github.com/coala/corobo/issues/236

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: answers/extraction.py [hunk 1]
```diff
@@ -89,7 +90,17 @@ def parse_docs():
...
     Parse all documentation files and store information in DATA
     """
+    setup_docs()
     for files in os.listdir(get_abs_path('coala/docs/Developers')):
         rst = parse_rst(get_abs_path('coala/docs/Developers/' + files))
         extractor = Extractor(rst, files)
         rst.walk(extractor)
+
+
+def setup_docs():
```
Comment (Udayan12167) [inline answers/extraction.py]: So I think it should also get new commits.

**Comment by Udayan12167** [comment]
ack https://github.com/coala/corobo/pull/237/commits/bed50bf63453312200ba05b67c3243f23e5b58a6

**Comment by Udayan12167** [comment]
ack https://github.com/coala/corobo/pull/237/commits/b35e91b0f1cca607cc7e6d1ebbd5475462f9b2e1

File: answers/final.py [hunk 2]
```diff
@@ -7,9 +7,8 @@
 import nltk
 import spacy
 
-from .extraction import DATA, parse_docs
+from .extraction import parse_docs
```
Comment (Udayan12167) [inline answers/final.py]: Any reason why this isn't an absolute import?

**Comment by Udayan12167** [comment]
ack https://github.com/coala/corobo/pull/237/commits/317d77b088a6df7715b5061253e30b726dda3ae9

**Comment by Udayan12167** [comment]
ack https://github.com/coala/corobo/pull/237/commits/bc81dbd284039b2e3bc439433deab6eca394a493

**Comment by Udayan12167** [comment]
Weird gitmate bug.

**Comment by meetmangukiya** [comment]
reack ea07f6b2fe795a2d5b1d27bca603391f79df7002

**Comment by meetmangukiya** [comment]
reack 0c4cf36dab700b27b242ecdd3451874299cafd7a
reack 29003636f66db09e47feea85c5d0286109b821dd

ack  b5787623a80d60dcd043d29e7f50ab24f1b17b1b
ack 7f1b9ccafce6861d1364b5ba911547ad9b119fd3

**Comment by yukiisbored** [comment]
lgtm

**Comment by meetmangukiya** [comment]
ack b9a84aa9

**Comment by meetmangukiya** [comment]
@rultor merge

**Pull request merged**